client/src/modules/CrossCheckPairs/components/CrossCheckPairsTable/CrossCheckPairsTable.tsx

```diff
@@ -0,0 +1,53 @@
+import { Table, TablePaginationConfig } from 'antd';
+import { CrossCheckPairDto } from 'api';
+import { FilterValue, SorterResult } from 'antd/lib/table/interface';
+import {
+  CustomColumnType,
+  fields,
+  getCrossCheckPairsColumns,
+} from 'modules/CrossCheckPairs/data/getCrossCheckPairsColumns';
+
+export type Filters = Omit<typeof fields, 'score' | 'submittedDate' | 'reviewedDate'>;
+
+interface CustomSorterResult<RecordType> extends SorterResult<RecordType> {
+  column?: CustomColumnType<RecordType>;
+}
+
+export type Sorter<RecordType> = CustomSorterResult<RecordType> | CustomSorterResult<RecordType>[];
+
+type CrossCheckTableProps = {
+  loaded: boolean;
+  crossCheckPairs: CrossCheckPairDto[];
+  pagination: TablePaginationConfig;
+  onChange: (
+    pagination: TablePaginationConfig,
+    filters: Record<keyof Filters, FilterValue | null>,
+    sorter: Sorter<CrossCheckPairDto>,
+  ) => void;
+  viewComment: (value: CrossCheckPairDto) => void;
+};
+
+export const CrossCheckPairsTable = ({
+  loaded,
+  crossCheckPairs,
+  pagination,
+  onChange,
+  viewComment,
+}: CrossCheckTableProps) => {
+  if (!loaded) return null;
+
+  // where 800 is approximate sum of basic columns (GitHub, Name, etc.)
```

Comment:
:)

Reply:
it's old code. I want to try to redo this table with pro-table in the future.